Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/repair.h

```diff
@@ -2179,59 +2179,41 @@ std::size_t make_umbrella_manifold(typename boost::graph_traits<PolygonMesh>::ha
   return nb_new_vertices;
 }
 
+} // end namespace internal
+
 /// \ingroup PMP_repairing_grp
-/// duplicates all the non-manifold vertices of the input mesh.
+/// collects the non-manifold vertices (if any) present in the mesh. A non-manifold vertex `v` is returned
+/// via one incident halfedge `h` such that `target(h, pm) = v` for all the umbrellas that `v` apppears in
+/// (an <i>umbrella</i> being the set of faces incident to all the halfedges reachable by walking around `v`
+/// using `hnext = prev(opposite(h, pm), pm)`, starting from `h`).
 ///
-/// @tparam PolygonMesh a model of `HalfedgeListGraph` and `MutableHalfedgeGraph`
-/// @tparam NamedParameters a sequence of \ref pmp_namedparameters "Named Parameters"
+/// @tparam PolygonMesh a model of `HalfedgeListGraph`
+/// @tparam OutputIterator a model of `OutputIterator` with value type `boost::graph_traits<PolygonMesh>::halfedge_descriptor`
```

Comment:
the value type of an output iterator is void. You can say "accepting elements of type" "or `hedge_descriptor` can be put inside "